nntrainer/tensor/blas_interface.cpp

```diff
@@ -16,6 +16,10 @@
 
 #include <cmath>
 
+#if (BLAS_NUM_THREADS > 0) && (BLAS_NUM_THREADS < 9999)
+#define NUM_THREADS_ BLAS_NUM_THREADS
```

Comment:
I wonder do we need to define NUM_THREADS_ instead of directly use BLAS_NUM_THREADS?

Reply:
done. thanks.